clang/test/CodeGenSYCL/static-var-address-space.cpp

```diff
@@ -0,0 +1,35 @@
+
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -triple spir64-unknown-unknown-sycldevice -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+
+struct C {
+  static int c;
+};
+
+template <typename T>
+struct D {
+  static T d;
+};
+
+template <typename T>
+void test() {
+  // CHECK: @_ZZ4testIiEvvE1a = linkonce_odr addrspace(1) constant i32 0, comdat, align 4
+  static const int a = 0;
+  // CHECK: @_ZZ4testIiEvvE1b = linkonce_odr addrspace(1) constant i32 0, comdat, align 4
+  static const T b = T(0);
+  // CHECK: @_ZN1C1cE = external addrspace(1) global i32, align 4
+  C::c = 10;
+  const C struct_c;
+  // CHECK: @_ZN1DIiE1dE = external addrspace(1) global i32, align 4
+  D<int>::d = 11;
+  const D<int> struct_d;
+}
+
+template <typename name, typename Func>
+__attribute__((sycl_kernel)) void kernel_single_task(const Func &kernelFunc) {
+  kernelFunc();
+}
```

Comment:
Could you please use a function with `sycl_kernel` attribute from mock runtime header in CodeGenSYCL/Inputs/sycl.hpp, for example `kernel_single_task`